x-pack/legacy/plugins/monitoring/public/components/setup_mode/enter_button.tsx

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { EuiButton } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+export interface SetupModeEnterButtonProps {
+  enabled: boolean;
+  toggleSetupMode: (state: boolean) => void;
+}
+
+export const SetupModeEnterButton: React.FC<SetupModeEnterButtonProps> = (
+  props: SetupModeEnterButtonProps
+) => {
+  const [isLoading, setIsLoading] = React.useState(false);
+
+  if (!props.enabled) {
+    return null;
+  }
+
+  async function enterSetupMode() {
+    setIsLoading(true);
+    await props.toggleSetupMode(true);
+    setIsLoading(false);
+  }
+
+  return (
+    <div style={{ position: 'absolute', top: 15, left: 12, zIndex: 1 }}>
```

Comment:
Can we give this div the class `monSetupModeEnterButton__buttonWrapper` and instead of doing inline styling apply those properties on sass using Eui variables?

```
.monSetupModeEnterButton__buttonWrapper {
    position: absolute;
    top: $euiSize;
    left: $euiSizeM;
    z-index: 1; 
}
```